src/pcapi/repository/booking_queries.py

```diff
@@ -325,8 +334,8 @@ def find_eligible_bookings_for_offerer(offerer_id: int) -> List[BookingSQLEntity
         .all()
 
 
-def find_eligible_bookings_for_venue(venue_id: int) -> List[BookingSQLEntity]:
-    return _query_keep_only_used_or_finished_bookings_on_non_activation_offers() \
+def find_payment_eligible_bookings_for_venue(venue_id: int) -> List[BookingSQLEntity]:
+    return _query_keep_payment_eligible_bookings() \
```

Comment:
```suggestion
    return _query_keep_only_used_and_non_cancelled_bookings_on_non_activation_offers() \
              ._query_keep_only_bookings_of_past_events() \
```

Est-ce que ce ne serait pas plus clair ?
J'ai dû retourner dans le ticket jira pour comprendre ce qu'étaient les "eligible bookings"

Reply:
Il y a aussi la notion d'évènement passé.
On empile les règles de gestion dans le nom de la fonction, cela me paraissait trop.

Reply:
C'est pour ça que j'ai maintenu la séparation en deux fonctions que tu as fait
Mais sorti `_query_keep_only_used_and_non_cancelled_bookings_on_non_activation_offers` de ta fonction où il était masqué